spark/src/test/scala/org/apache/spark/sql/delta/skipping/clustering/IncrementalZCubeClusteringSuite.scala

```diff
@@ -230,7 +230,7 @@ class IncrementalZCubeClusteringSuite extends QueryTest
                 inputZCubeFiles = ClusteringFileStats(2, SKIP_CHECK_SIZE_VALUE),
                 inputOtherFiles = ClusteringFileStats(4, SKIP_CHECK_SIZE_VALUE),
                 inputNumZCubes = 1,
-                mergedFiles = ClusteringFileStats(6, SKIP_CHECK_SIZE_VALUE),
+                mergedFiles = ClusteringFileStats(4, SKIP_CHECK_SIZE_VALUE),
```

Comment:
Im guessing this is also same reason as above

Reply:
Yes, this is fixing the test bug introduced in https://github.com/delta-io/delta/pull/3793/files#r1815830851